src/components/selectionBoxLayer.ts

```diff
@@ -2,6 +2,7 @@
 
 module Plottable {
 export module Components {
+  enum PropertyMode { VALUE, PIXEL };
```

Comment:
This has already been defined in `guideLineLayer`, hence can be removed from here. 
Even better, extract it from there, put it into `interfaces` or `component` so we can find it easier now.